EarlGrey/Action/GREYActions.h

```diff
@@ -285,6 +291,12 @@
 /** Shorthand macro for GREYActions::actionForMultipleTapsWithCount: with count @c 2. */
 GREY_EXPORT id<GREYAction> grey_doubleTap(void);
 
+/**
+ *  Shorthand macro for
+ *  GREYActions::actionForMultipleTapsWithCount: with count @c 2 and @c point.
+ */
+GREY_EXPORT id<GREYAction> grey_doubleTapAtPoint(CGPoint point);
```

Comment:
I think the cheatsheet.html should be updated to include `grey_doubleTapAtPoint` so the next time the png/pdf are generated, the new API is there.